### PR TITLE
changed `threads` default

### PR DESCRIPTION
**Pull request by mjurbanski-reef**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 * Better help text for --corsRules
+* if `--threads` is not explicitly set number of threads is no longer guaranteed to be 10
```
Comment (jsuchan-reef) [inline CHANGELOG.md:15]: btw where/how is this guarantee violated? The default value is still 10, no?
```suggestion
* if `--threads` is not explicitly set, number of threads is no longer guaranteed to be 10 
```

**Review by jsuchan-reef — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mjurbanski-reef) [reply CHANGELOG.md]: the default is 10, but the idea is that in future it maybe very well automatically adjusted and we don't want users relying on the fact it is 10 or, in fact, any specific number.

Comment (jsuchan-reef) [reply CHANGELOG.md]: then I'd assume this note to appear in the changelog when such change will be in effect.

**Comment by mjurbanski-reef** [comment]
merged upstream couple months ago

**Pull request closed without merging**